department-of-veterans-affairs/abd-vro
# Logging event body as a formatted json message and a flag

## What was the problem?
Body was printed as string with an message.

Associated tickets or Slack threads:
- #1683 

## How does this fix it?
Printing event message as json and `ReceivedMessageEventBody` prepended.

```
2023-07-20 14:29:10.775  INFO 65698 --- [tion-classified] nts-contention-classified-saveToDb-route : ReceivedMessageEventBody: {"status":200,"event":"test","eventDetails": "EventDetailsMessage","notifiedAt":"2023-07-15T00:00:00"}
```

## How to test this PR
- See above log message (tested this locally by sending a message through rabbitmq)
- Plan is to deploy this to dev
- Send messages using rabbitmq or Kafka
- See messages in Datadog
- Optionally configure Datadog to Slack and see the messages in Slack.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
